### PR TITLE
Add Gubernator row to the instance utilization dashboard

### PR DESCRIPTION
**Pull request by douglascamata**
Dashboard preview: https://grafana.stage.devshift.net/goto/cWMxVur4z?orgId=1

As we don't have yet a detailed dashboard for Gubernator, it doesn't offer a dashboard link and is adapted to this.

### CODE REVIEW

File: observability/dashboards/rhobs-instance-utilization-overview.libsonnet [hunk 1]
```diff
@@ -710,6 +719,121 @@ function() {
...
+        g.row('Gubernator Overview')
+        .addPanel(
+          g.panel('Rate of gRPC requests', 'Shows count of gRPC requests to gubernator') +
+          g.queryPanel(
+            [
+              'sum(rate(gubernator_grpc_request_counts{namespace="$namespace",job=~"$job"}[$__rate_interval])) by (namespace,job,pod)',
+            ],
+            [
+              'gRPC requests {{pod}}',
+            ]
```
Comment (thibaultmg) [inline observability/dashboards/rhobs-instance-utilization-overview.libsonnet:732]: Shouldn't we add the g.stack on the panels?

File: observability/dashboards/rhobs-instance-utilization-overview.libsonnet [hunk 2]
```diff
@@ -710,6 +719,121 @@ function() {
...
+              'pod restart count {{pod}}',
+            ]
+          ) +
+          g.addDashboardLink(am.title)
+        )
+        .addPanel(
+          g.panel('Network Traffic') +
+          g.queryPanel(
+            [
+              'sum by (pod) (rate(container_network_receive_bytes_total{namespace="$namespace", pod=~"observatorium-alertmanager.*"}[$interval]))',
```
Comment (coleenquadros) [inline observability/dashboards/rhobs-instance-utilization-overview.libsonnet:824]: here too - gubernator

File: observability/dashboards/rhobs-instance-utilization-overview.libsonnet [hunk 3]
```diff
@@ -710,6 +719,121 @@ function() {
...
+            [
+              'peer queue size {{pod}}',
+            ]
+          )
+        )
+        .addPanel(
+          g.panel('Memory Used', 'Memory working set') +
+          g.queryPanel(
+            [
+              '(container_memory_working_set_bytes{container="observatorium-alertmanager", namespace="$namespace"})',
```
[the remark this answers is not in the document]
Comment (douglascamata) [reply observability/dashboards/rhobs-instance-utilization-overview.libsonnet:786]: Nice one. That's body copy-pasta from me. 😬

**Comment by douglascamata** [comment]
@thibaultmg can re-review please? Fixed all the issues :)

**Review by thibaultmg — APPROVED** [review]
LGTM!

**Pull request merged**